frontend/benefit/applicant/src/components/applications/forms/application/step5/employeeView/EmployeeView.tsx

```diff
@@ -59,29 +63,30 @@ const EmployeeView: React.FC<EmployeeViewProps> = ({
         withoutDivider
       >
         <$GridCell $colSpan={3}>
-          <$ViewFieldBold>
-            {t(
-              `${translationsBase}.employee.fields.paySubsidyGranted.${
-                data.paySubsidyGranted ? 'yes' : 'no'
-              }`
-            )}
-            {data.apprenticeshipProgram && (
-              <$ViewField>{`, ${data.paySubsidyPercent || ''} %`}</$ViewField>
-            )}
-          </$ViewFieldBold>
+          {data.paySubsidyGranted && (
+            <$ViewFieldBold>
+              {t(
+                `${translationsBase}.employee.fields.paySubsidyGranted.${
+                  data.paySubsidyGranted ? 'yes' : 'no'
+                }`
+              )}
+              {data.apprenticeshipProgram && (
+                <$ViewField>{`, ${data.paySubsidyPercent || ''} %`}</$ViewField>
```

Comment:
I saw a separate bug ticket about this comma here. Is it intentional?
```suggestion
                <$ViewField>{`${data.paySubsidyPercent || ''} %`}</$ViewField>
```

Reply:
the order was completely wrong. maybe it was copypaste or something, but now it looks correctly